source/feature-manager.tsx

```diff
@@ -274,11 +274,13 @@ const add = async (url: string, ...loaders: FeatureLoader[]): Promise<void> => {
 			void setupPageLoad(id, details);
 		}
 
-		document.addEventListener('turbo:render', () => {
-			if (!deduplicate || !select.exists(deduplicate)) {
-				void setupPageLoad(id, details);
-			}
-		});
+		for (const eventType of ['turbo:render', 'soft-nav:render']) {
```

Comment:
Both of these events are called when clicking one of the repos tabs, for example when you click on Issues. This means you're running every feature twice on every page load.

Also, neither one is called when you:

1. Open https://github.com/refined-github/refined-github/tree/main/.github/workflows
2. Go up one folder
3. Go back in browser history ❌ 
4. Go forward in browser history ❌

Reply:
> I might push a fix specific to this feature

I "fixed the duplication" by using `[title="More options"]:not(:has(+ .rgh-download-folder-button))`, but this means that the old button is left in place, pointing to the old folder. 🥲 

Let's move the discussion to:

- https://github.com/refined-github/refined-github/issues/6554